cmd/otel-agent/config/agent_config.go

```diff
@@ -25,6 +25,36 @@ import (
 	"go.opentelemetry.io/collector/service"
 )
 
+type logLevel int
+
+const (
+	off logLevel = iota
+	critical
+	err
+	warn
+	info
+	debug
+	trace
```

Comment:
The order seems reverse of general order for levels with ex: critical should have a higher number than info

Reply:
The order is increasing from least verbose to most verbose. Not sure I understand the concern here? This allows us to compare the log values directly.

Reply:
See https://github.com/uber-go/zap/blob/f87c7be57ccc7ab7a0aa3fbc12398369fe3a61c3/zapcore/level.go#L37 or https://opentelemetry.io/docs/specs/otel/logs/data-model/#severity-fields
info is generally lower number than critical ?